tests/package.json

```diff
@@ -5,9 +5,11 @@
   "description": "This is Taqueria testing project includes unit, integration and e2e tests",
   "main": "index.js",
   "scripts": {
-    "test:unit": "deno test unit/ --allow-all --unstable",
-    "test:unit:debug": "deno test unit/ --allow-all --unstable --inspect-brk",
-    "test:unit:with-coverage": "deno test unit/ --allow-all --unstable --coverage=coverage_report && deno coverage coverage_report && deno coverage coverage_report --lcov > coverage_profile.lcov",
+    "test:unit": "deno test unit/ --allow-all --unstable && jest --config jest.config.unit.ts --no-cache",
```

Comment:
@sinapsist a suggestion to avoid duplication would be to set `test:unit` to `npm run test:unit:deno && npm run test:unit:jest`?

Reply:
Good point. Updating.